eng/Versions.props

```diff
@@ -19,7 +19,8 @@
     <SdkBandVersionForWorkload_FromRuntimeVersions>$(SdkBandVersion)$(WorkloadVersionSuffix)</SdkBandVersionForWorkload_FromRuntimeVersions>
     <!-- Set assembly version to align with major and minor version,
          as for the patches and revisions should be manually updated per assembly if it is serviced. -->
-    <AssemblyVersion>$(MajorVersion).$(MinorVersion).0.0</AssemblyVersion>
+    <!-- Temporarily hard-code to 9.0.0.0. We can't version forward the Assembly Version without updating the TFM in dotnet/runtime. -->
+    <AssemblyVersion>9.0.0.0</AssemblyVersion>
```

Comment:
@jkoritzinsky do we need to do this again like last year?

Reply:
Is there a need for a doc that describes the key updates?

Reply:
I'm unaware of any official docs. We do have an internal OneNote with instructions for us runtime infra folks.

For as long as I've been in the main person in charge of this, we have been copying what we did the previous year, then adapt anything that either fails on the CI or that we know has changed since the last time.

Reply:
It would be good to document the instructions publicly so that someone running a downstream project could adopt different patterns if they wanted to.

Reply:
Nice! Looking.

Reply:
Reverting this.